tests/test_file_parser.py

```
@@ -1,493 +1,113 @@
-import aiohttp
-from aioresponses import aioresponses
+from path2dream_processors.file_parser import APIBasedFileParser
 
-from path2dream_processors.file_parser import APIBasedFileParser, FileType
 
+@pytest.mark.asyncio
+async def test_real_audio_parsing():
+    """Test parsing real audio file with OpenAI Whisper API."""
+    if not os.getenv('OPENAI_API_KEY'):
+        pytest.fail("OPENAI_API_KEY environment variable is required for this test")
+    
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)